### PR TITLE
[#1655] fixed k=x generation if k is set in parser options

### PR DESCRIPTION
**Pull request by cdietrich**
[#1655] fixed k=x generation if k is set in parser options
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2 [hunk 1]
```diff
@@ -27,9 +27,9 @@ XtextGeneratorLanguage {
 
 	// The antlr parser generator fragment.
 	fragment = parser.antlr.XtextAntlrGeneratorFragment2 auto-inject {
-	//  options = {
-	//      backtrack = true
-	//  }
+	  options = {
+	     k = 1
```
Comment (cdietrich) [inline org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2:31]: i hope i did the math correct. that do you think @szarnekow

Comment (szarnekow) [reply org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2:31]: Antlr agrees.

**Review by szarnekow — APPROVED** [review]
LGTM

File: org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2 (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2:31]: how/where can i see this?

Comment (szarnekow) [reply org.eclipse.xtext.testlanguages/src/org/eclipse/xtext/testlanguages/noJdt/NoJdtTestLanguage.mwe2:31]: The generated code didn't change apart from line numbers in comments - no other LA related changes.

**Pull request merged**